__tests__/components/editor/property/LiterTypeLabel.test.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2021 Stanford University see LICENSE for license
+
+import React from "react"
+import { createStore, renderComponent } from "testUtils"
+import { createState } from "stateUtils"
+import LiteralTypeLabel from "components/editor/property/LiteralTypeLabel"
+
+describe("<LiteralTypeLabel />", () => {
```

Comment:
Can you fix the name of this file? (LiterTypeLabel.test.js)

Reply:
fixed